lib/tasks/test.rake

```diff
@@ -67,4 +67,98 @@ namespace :test do
     puts '@notifier.ping normal (Slack)'
     @notifier.ping('Testing .ping polymorphism - this should go to Slack')
   end
+
+  desc 'Evaluate all TodoOrDie calls in the codebase'
+  task todo_or_die: :environment do
```

Comment:
Interesting, I'm a little unsure about this approach. Have you tried something like this before? 

Could we just load the whole application (eager_load) and then see what todo errors are generated? That way we don't need to use eval() or add the unparser dependency. It also seems a lot less complex

```ruby
namespace :test do
  desc 'Evaluate all TodoOrDie calls by loading the application'
  task todo_or_die: :environment do
    # Force eager loading to trigger all TodoOrDie calls
    Rails.application.eager_load!
   
  # could rescue although I think just letting the exception happen should work
  # rescue TodoOrDie::OverdueTodo => e
  #   puts "Overdue TODO found: #{e.message}"
  #   exit(1) if ENV['CI']
  end
end
```